content/en/docs/tasks/run-application/rolling-update-replication-controller.md

```diff
@@ -65,18 +71,22 @@ Replication controller configuration files are described in
 
 ### Examples
 
-    // Update pods of frontend-v1 using new replication controller data in frontend-v2.json.
-    kubectl rolling-update frontend-v1 -f frontend-v2.json
+```shell
+# Update pods of frontend-v1 using new replication controller data in frontend-v2.json.
+kubectl rolling-update frontend-v1 -f frontend-v2.json
 
-    // Update pods of frontend-v1 using JSON data passed into stdin.
-    cat frontend-v2.json | kubectl rolling-update frontend-v1 -f -
+# Update pods of frontend-v1 using JSON data passed into stdin.
+cat frontend-v2.json | kubectl rolling-update frontend-v1 -f -
+```
 
 ## Updating the container image
 
 To update only the container image, pass a new image name and tag with the
 `--image` flag and (optionally) a new controller name:
 
-    kubectl rolling-update NAME [NEW_NAME] --image=IMAGE:TAG
+```shell
+kubectl rolling-updatea NAME NEW_NAME --image=IMAGE:TAG
+```
```

Comment:
- `kubectl rolling-update` is deprecated? Is there an alternative?
- check the spelling of rolling-update/rolling-updatea

Reply:
Good spot!
I might reword in a separate PR. I like PRs that do one thing.